src/main/java/org/jenkinsci/plugins/awsdevicefarm/AWSDeviceFarmTestResult.java

```diff
@@ -80,11 +84,12 @@ public AWSDeviceFarmTestResult(AbstractBuild<?, ?> build, Run run) {
      * Map Device Farm results to Jenkins results.
      */
     static {
-        resultMap.put("PASSED", Result.SUCCESS);
-        resultMap.put("WARNED", Result.UNSTABLE);
-        resultMap.put("FAILED", Result.FAILURE);
-        resultMap.put("SKIPPED", Result.FAILURE);
-        resultMap.put("ERRORED", Result.FAILURE);
+        resultMap.put(ExecutionResult.PASSED.toString(), Result.SUCCESS);
+        resultMap.put(ExecutionResult.WARNED.toString(), Result.UNSTABLE);
+        resultMap.put(ExecutionResult.FAILED.toString(), Result.FAILURE);
+        resultMap.put(ExecutionResult.SKIPPED.toString(), Result.FAILURE);
```

Comment:
Can you store the ExecutionResult enum value in the map instead of the string representation?